templates/zerver/help/change-date-time.md

```diff
@@ -0,0 +1,20 @@
+# Change the date and time format
+![Display time in 24-hour format](/static/images/help/24-hour.png)
+
+By default, messages in Zulip are displayed with a 12-hour timestamp. If you are used to seeing time displayed in a 24-hour format, you can change the Zulip timestamp format from a 12-hour format to a 24-hour format. You may want to do this to avoid confusion between A.M. and P.M. hours.
+
+You can easily change the date and time format your messages are displayed in by following a few steps.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar.
+2. Select **Settings** from the dropdown menu that appears.
+
+    ![Settings](/static/images/help/settings.png)
+
+3. Scroll down to the **Display Settings** section (after the **My Account** section) to view the message display options.
+
+    ![Message Display](/static/images/help/message-display-settings.png)
```

Comment:
Put a blank line after here.  This will make the code easier to read and fix some strange rendering issues in Chrome.